src/analysis/normed_space/inner_product.lean

```diff
@@ -1481,6 +1481,14 @@ def inner_product_space.is_R_or_C_to_real : inner_product_space ℝ E :=
   ..has_inner.is_R_or_C_to_real 𝕜 E,
   ..normed_space.restrict_scalars ℝ 𝕜 E }
 
+instance pi_Lp.inner_product_space_real {ι : Type*} [fintype ι] (f : ι → Type*)
+  [Π i, inner_product_space 𝕜 (f i)] : inner_product_space ℝ (pi_Lp 2 one_le_two f) :=
+inner_product_space.is_R_or_C_to_real 𝕜 _
+
+instance pi_Lp.is_scalar_tower {ι : Type*} [fintype ι] (f : ι → Type*)
+  [Π i, inner_product_space 𝕜 (f i)] : is_scalar_tower ℝ 𝕜 (pi_Lp 2 one_le_two f) :=
+restrict_scalars.is_scalar_tower _ _ _
```

Comment:
I think this belongs in `topology/metric_space/pi_Lp.lean`, and should produce the result `is_scalar_tower 𝕜 𝕜' (pi_Lp p hp α)` for all `𝕜` that meet the requirements of `pi_Lp.normed_space`. The proof should just be `pi.is_scalar_tower`.

Reply:
#18577